swf/src/types/color_transform.rs

```diff
@@ -77,59 +62,59 @@ impl Default for ColorTransform {
     }
 }
 
-impl Mul for ColorTransform {
-    type Output = Self;
+impl Mul for &ColorTransform {
```

Comment:
Actually now that this is `Copy`, can this change back to impl `for ColorTransform`?